src/BBLibc/BBLibc.h

```diff
@@ -0,0 +1,39 @@
+#ifndef BBLIBC_H
+
+#define BBLIBC_H
+
+#include <bld_system.h>
+#include <bld_python.h>

```

Comment:
I suggest to remove Python include and use forward declaration `struct PyObject`. It can resolve https://github.com/smartblade/BldMystery/pull/5#discussion_r1755127785.